install.sh

```diff
@@ -49,59 +66,46 @@ if psql -lqt | cut -d \| -f 1 | grep -qw $DBNAME; then
 
 else
 
-   #Create Postgresql Pre-reqs
-   sudo postgres <<EOF
-   psql -c "CREATE USER $DBUNAME WITH PASSWORD '$DBPASS' CREATEDB;"
-   createdb -O$DBUNAME -Eutf8 $DBNAME
-   echo "Postgres user and database '$1' created."
+#Create Postgresql Pre-reqs
+sudo su postgres <<EOF
+psql -c "CREATE USER $DBUNAME WITH PASSWORD '$DBPASS' CREATEDB;"
+createdb -O$DBUNAME -Eutf8 $DBNAME
+echo "Postgres user '$DBUNAME' and database '$DBNAME' created."
 EOF
 
 fi
 
-# TODO
-# echo "Creating web server user"
-# sudo useradd $DBUNAME
-
-echo "Creating clone of git repo"
-sudo mkdir -p $APP_PATH
-# sudo chown $DBUNAME:$DBUNAME $APP_PATH # TODO
-# sudo -u $DBUNAME git clone $GIT_PATH $APP_PATH # TODO
-git clone $GIT_PATH $APP_PATH
 
 echo "Installing required application packages...(if needed)"
 
 yes | pip3 install virtualenv
 
-pushd $APP_PATH
+cd $APP_PATH
 if [ ! -d "$APP_PATH/venv" ]; then
-  # sudo -u $DBUNAME python3 -m virtualenv venv # TODO
-  python3 -m virtualenv venv
+  sudo python3 -m virtualenv venv
 fi
 
 source venv/bin/activate
 yes | pip3 install -r requirements.txt
 
-echo "Writting application config"
-cat << EOF > $APP_PATH/HardwareCheckout/config.py
+deactivate
+
+#Configure Application
+sudo cat << EOF > $APP_PATH/HardwareCheckout/config.py
 #!/usr/bin/env python3
 db_path = 'postgresql+psycopg2://$DBUNAME:$DBPASS@127.0.0.1:5432/$DBNAME'
-ssl_config = {
-  'certfile':'',
-  'keyfile':''
-  }
```

Comment:
Please add this back. We need to to support the new SSL config.

Reply:
Added